python/taichi/aot/__init__.py

```diff
@@ -7,3 +7,15 @@
 from taichi.aot.conventions.gfxruntime140 import GfxRuntime140
 from taichi.aot.module import Module
 from taichi.aot.record import *
+
+import taichi as ti
+
+_aot_kernels = []
```

Comment:
aha can we move this impl inside a file so that we can keep __init__.py short (like just imports) ;) ?